### ISSUE TITLE
Display Posts in Sortable Table

### ISSUE THREAD

**Issue by iamtakashi** [WordPress/gutenberg#25503]
Related: #25501 #25502 

It's less common than the related cases above, but what if the user wants to display posts by a Query Loop block in a sortable table?

![Frame 68](https://user-images.githubusercontent.com/908665/93767993-a2d89280-fc10-11ea-8a0a-9a72aa98dbb7.jpg)

**Comment by ntsekouras** [comment]
Hey @iamtakashi I think this is a quite complex proposal that includes a whole new component/block ( Sortable table ) and many nuances to that, like pagination etc. My above comment I believe applies for the `Carousel and Slider` issues as well. If it comes to that, the first steps would be to implement the above blocks and then implement them as a block variation probably.

I'm curious though if the table case would be useful enough. Carousel and Slider would be really helpful for sure in my opinion.

Regarding the issue here for inline posts: https://github.com/WordPress/gutenberg/issues/25505, I think this can be achieved with css. It's on the next steps to support display of items as a vertical list or grid(inline), but that would be just simple styling, at least for start.

There are many questions to be answered yet, so more iterations and feedback will show us the way :).

**Comment by iamtakashi** [comment]
Hi @ntsekouras, thanks for sharing your thoughts and the steps that'd likely take place. Much appreciated.

> If it comes to that, the first steps would be to implement the above blocks and then implement them as a block variation probably.

Just to make sure I'm understanding correctly what you mean, are you saying _if_ Gutenberg allows to have Carousel or Slider at some point, those would be separate blocks, not with the query loop block?

> I'm curious though if the table case would be useful enough. Carousel and Slider would be really helpful for sure in my opinion.

Yes, carousel and slider would be much more popular than this for sure. I should've noted that, like the inline posts #25505, this sortable table was just an example of an unusual case that some pattern makers might want to achieve. This was more of a question of how one could do it, and I didn't really mean to advocate Gutenberg to include the sortable table out of the box :)

> There are many questions to be answered yet, so more iterations and feedback will show us the way :).

Yes :) I'm very excited about how the query loop block evolve going forward.

**Comment by ntsekouras** [comment]
>Just to make sure I'm understanding correctly what you mean, are you saying if Gutenberg allows to have Carousel or Slider at some point, those would be separate blocks, not with the query loop block?

I feel like they should be separate blocks because a common use case of Slider is with just photos and not necessarily with posts. It could be with taxonomies and their image.. Same goes for Carousel.

**Comment by iamtakashi** [comment]
> a common use case of Slider is with just photos and not necessarily with posts. It could be with taxonomies and their image.. Same goes for Carousel.

Yes, there are sliders and carousels for just purely images, but it's fairly common to see the ones displaying posts or pages.

**Comment by ntsekouras** [comment]
Closing this one as there hasn't been any traction since 2020. Feel free to reopen if I'm missing something.
